SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -174,7 +176,11 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 
 		// No path in SpatialOS should contain a PIE prefix.
 		FString TempPath = Actor->GetFName().ToString();
+#if ENGINE_MINOR_VERSION >= 26
+		GEngine->NetworkRemapPath(NetDriver->GetSpatialOSNetConnection(), TempPath, false /*bIsReading*/);
```

Comment:
Minor: Since we are doing `/*bIsReading*/` here which I also prefer, we should do it in all of the other `NetworkRemapPath` calls.